client/me/help/help-contact/index.jsx

```diff
@@ -578,7 +579,7 @@ const HelpContact = React.createClass( {
 		//    requests are sent to the language specific forums (for popular languages)
 		//    we don't tell the user that support is only offered in English.
 		const showHelpLanguagePrompt =
-			( olark.locale !== i18n.getLocaleSlug() ) &&
+			( currentUserLocale !== i18n.getLocaleSlug() ) &&
```

Comment:
What's the condition where this will be true? I tried triggering it but it's hard to trace down exactly where `localeSlug` is set in the `i18n` code. It seems possible that `currentUserLocale` will always be the same as `i18n.getLocaleSlug()` but I'm not sure?